gammapy/image/utils.py

```diff
@@ -387,7 +387,7 @@ def coordinates(image, world=True, lon_sym=True, radians=False):
         y, x = np.indices(image.data.shape, dtype='int32') + 1
 
     wcs = WCS(image.header)
-    lon, lat = wcs.wcs_pix2world(x, y, 1)
+    lon, lat = wcs.wcs_pix2world(x, y, 0)
```

Comment:
I don't know when I'll get to this. 
For now I've put an entry describing what our pixel coordinate convention is in the docs:
https://gammapy.readthedocs.org/en/latest/development/index.html#pixel-coordinate-convention